framework/requirements/YiiRequirementChecker.php

```diff
@@ -112,7 +112,7 @@ function check($requirements)
      */
     function checkYii()
     {
-        return $this->check(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'requirements.php');
+        return $this->check(__DIR__ . DIRECTORY_SEPARATOR . 'requirements.php');
```

Comment:
Should not be modified like that because requirements checker requirements are lower than framework's. It should be able to say that requirements aren't met.